packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/android/gettingatoken.md

```diff
@@ -0,0 +1,24 @@
+```kotlin
+private suspend fun handleResponse(response: IdxResponse) {
```

Comment:
I find it easier to implement the things I see in tutorials like this if they include imports.

I think it's worth adding the few extra lines to increase clarity.

Reply:
Sounds good... and the imports would be?
Seriously, leave a comment with the imports and I'll add them. Thanks.

Reply:
I think it's just:
- `import com.okta.idx.kotlin.dto.IdxRemediation.Type.ISSUE`
- `import com.okta.idx.kotlin.dto.IdxResponse` 
- `import com.okta.idx.kotlin.client.IdxClientResult`